packages/light-client/test/browser

```diff
@@ -0,0 +1 @@
+./unit
```

Comment:
What does this do?

Reply:
Symlink `unit` directory to the `browser`